pkgs/development/python-modules/nimfa/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, numpy
+, scipy
+, matplotlib
+, pytest
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
no `name`

Reply:
Python packages shouldn't have a `name`?

Reply:
`buildPythonPackage` will automatically set the name according to `pname` and `version`.